src/index.js

```diff
@@ -1,14 +1,70 @@
 // import _ from 'lodash';
 import './style.css';
 
-// function component() {
-//   const element = document.createElement('div');
+// const gameID = 'AsHkmrIRRRVDeNZNfO3f';
+const playername = document.getElementById('playername');
+const playerscore = document.getElementById('playerscore');
+const scoresubmit = document.getElementById('scoresubmit');
 
-//   // Lodash, now imported by this script
-//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-//   element.classList.add('hello');
+async function getScores() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/{$gameID}/scores/');
+  if (response.ok) {
+    const data = await response.json();
+    const ul = document.getElementById('list');
+    ul.innerHTML = '';
+    data.result.forEach((score) => {
+      const li = document.createElement('li');
+      li.innerHTML = `${score.user}: ${score.score}`;
+      ul.appendChild(li);
+    });
+  } else {
+    // console.log('Error');
+  }
+}
 
-//   return element;
+async function postScores(name, score) {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/{$gameID}/scores/', {
+    method: 'POST',
+    headers: {
```

Comment:
- [ ] Please consider using `arrow functions` (`() => {}`) instead of the `function` keyword for defining functions, as specified in the project requirements. `Arrow functions` provide a more concise syntax and lexical scoping of `this`, which can improve code readability and maintainability. See below for more details on project requirements:
<img width="537" alt="Screenshot 2023-06-30 at 16 17 01" src="https://github.com/cancelei/Leaderboard/assets/107192076/fe2837b4-0d7c-415e-bdf6-6c6e252e86d1">